package.json

```diff
@@ -7,11 +7,23 @@
   "homepage": "https://algorithmia.com/",
   "license": "The MIT License (MIT)",
   "scripts": {
+    "apidocs:build": "cd api-docs gem install bundler && bundle install && bundle exec rake build",
+    "apidocs:setup": "git submodule init && git submodule update && cd api-docs && rvm install 2.3.0 && gem install bundle && bundle install",
+    "build": "npm run apidocs:build && npm run synapse:build && npm run devcenter:build",
+    "devcenter:build": "./build.sh",
+    "devcenter:dev": "bundle exec jekyll serve --port 4001",
+    "devcenter:enterprise": "bundle exec jekyll serve --config _config.yml,_config-dev.yml,_config-enterprise.yml",
+    "devcenter:public": "bundle exec jekyll serve --config _config.yml,_config-dev.yml",
     "e2e": "./node_modules/.bin/wdio wdio.conf.js",
     "e2e:cloud": "E2E_RUN_IN_CLOUD=true npm run e2e",
     "e2e:tunnel": "E2E_BASE_URL=http://localhost:3000 E2E_USE_TUNNEL=true npm run e2e:cloud",
     "e2e:local": "E2E_BASE_URL=http://localhost:3000 npm run e2e",
-    "start": "NODE_ENV=production node ./server/index.js",
+    "setup": "npm run apidocs:setup && npm run apidocs:build && npm run synapse:setup && gem install bundler && bundle install && npm install",
```

Comment:
How come we have to run `apidocs:build` as part of the setup script?

Reply:
So that we have the `build/` directory set up so the express server can find those files.

I figured since the api-docs repo doesn't get updated much, making sure those files were built during setup was generally more convenient than adding an extra step.